### DISCUSSION TITLE
Welcome to Conreq Discussions!

### DISCUSSION THREAD

**Discussion by Archmonger** [Archmonger/Conreq#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MPulse25** [comment]
Hi, the search function is not working. Doesn't matter what movie or TV show title I search for it always returns no results were found. Is this a known bug or an issue with my installation/setup?

**Reply by Archmonger** [reply]
Search will not work if sonarr and radarr are disabled, or if your sonarr/radarr setting are incorrect.

**Comment by TheEscPilot** [comment]
Great work! App was easy to configure. When connecting my Radarr + Sonarr, I entered the data (API and IP:Port) for both, and both failed initially and was unable to pull quality/folder information (using Chrome). Refreshed the page after toggling each setting and now they're working perfect. 

A few Quality of Life things that would be nice
Set the the API key's to be obfuscated when the field isn't selected
Set an option to prefill user account data for new user accounts

Love the interface and the recommendations. It would be amazing to be able to select multiple recommendations and choose to download all from the recommendations page.

Again Great work you magic maker.

**Comment by RoryNesbitt** [comment]
Love the app so far, however before I start adding users it would be great to have the ability to have requests wait for approval. And more so if when approving I could choose which library to add it too

**Reply by Archmonger** [reply]
Hey Rory, 

I agree! Request approval is planned for v1.

Feel free to check out GitHub Projects to see what other features we have planned!

**Comment by hackmonker** [comment]
Is it possible to use plex or jellyfin as library source instead of just sonarr or radarr?

**Reply by Archmonger** [reply]
I currently don't have plans to directly integrate with Plex libraries.

The current content discovery code is going to become a plugin soon, so it'll be easily modifiable if someone wanted to integrate plex into it.

Also a "Plex-Like" plugin for streaming content will be coming within V1.
